process/templates/WG_NAME_graduation_stage.md

```diff
@@ -1,11 +1,41 @@
 ## Working Group graduation application
 
+### WG has met all Incubating requirements
+  * "link to Incubating PR"
+
+### List WG Chair(s) and or Vice Chair
+The WG must have a minimum of 2 Chair(s) or 1 Chair and 1 Vice Chair
+
+  * "name, affiliation, GitHUb ID"
+
```

Comment:
```suggestion
```

This is an interesting idea for the future, but for now we're going to not add new requirements until after we land these docs.